client/get.go

```
@@ -123,14 +129,17 @@ func retrieveTinkKeyset(keyID string) ([]byte, error) {
 	return tinkKeysetInBytes, nil
 }
 
-func retrieveTinkKeysetInfo(keyID string) (string, error) {
+func retrieveTinkKeysetInfo(keyID string, getFromNetwork bool) (string, error) {
 	if !isIDforTinkKeyset(keyID) {
 		return "", fmt.Errorf("this knox identifier is not for tink keyset")
 	}
 	// get the primary and all active versions of this knox identifier.
-	primaryAndActiveVersions, err := cli.GetKey(keyID)
-	if err != nil {
-		return "", fmt.Errorf("error getting key: %s", err.Error())
+	var primaryAndActiveVersions *knox.Key
+	var err error
+	if getFromNetwork {
+		primaryAndActiveVersions, err = cli.NetworkGetKey(keyID)
+	} else {
+		primaryAndActiveVersions, err = cli.GetKey(keyID)
 	}
```

this error is never handled.